chinese-checkers/Views/MainPage.xaml.cs

```diff
@@ -92,13 +100,15 @@ protected override void OnNavigatedTo(NavigationEventArgs e)
         protected override void OnNavigatedFrom(NavigationEventArgs e)
         {
             base.OnNavigatedFrom(e);
+            
 
             IsPaused = true;
         }
 
         public void CreateGameSession()
         {
             gs = new GameSession(locations, NumberOfAI, PlayerCharacter);
+            gs.CurrentlyPlaying.Highligh = true;
```

Comment:
To make a character highlighted for the first time when the player starts a new game.